common/fw-update-helper.cpp

```diff
@@ -179,7 +179,8 @@ namespace rs2
                      "and restart the realsense-viewer");
         }
         _done = true;
-        // need to find a way to update the fw version field in the viewer
+        // update the fw version field in the viewer
```

Comment:
`Restart the device to reconstruct with the new version information`?

Reply:
Changed